### ISSUE TITLE
Create Docker build for MedStar

### ISSUE THREAD

**Issue by kbertodatti** [chronic-care/mcc-project#263]
This task to create a Docker build for MedStar is low priority at this time and may need input from MedStar before we can proceed, e.g. what kind of config parameter they need for their Cerner test system.  MedStar may join one of our Wed tech calls over the next few weeks, but likely won't be ready to test for several weeks, or even several months.
